solutions/recursion/recursive-series.js

```diff
@@ -0,0 +1,33 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+	idx++;
+	return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+function s(n){
```

Comment:
@prateek-mehra ,even though in the example provided they used `S` for function name , it's better to use a better name for the function  instead of `s`

Reply:
@LVSsandeepVarma can you please suggest another function name? Because my logic was that it would be better for the students to understand because of how it is defined in the problem statement. Moreover, thinking of other names also provided ambiguity with respect to what the function actually does, according to me.

Reply:
@prateek-mehra  like `Series`

Reply:
@LVSsandeepVarma Done.